core/services/src/main/java/org/exoplatform/services/cms/link/LinkManager.java

```diff
@@ -192,5 +192,17 @@ default boolean isFileOrParentALink(Session session, String path) {
    * @throws Exception
    */
   public void updateSymlink(Node link) throws Exception;
-  
+
+  /**
+   * get the link of the document inside the folder
+   *
+   * @param contentUUID Jcr uuid of the file
+   * @param folderPath path of the target folder
+   * @param workspace target workspace
+   * @param nodeType type of current node
+   */
+  default Node getLinkInFolder(String contentUUID, String folderPath, String workspace, String nodeType) {

```

Comment:
Rename the function and do not add the NodeType inside it. In this service we just manage Symlinks
```suggestion
  default Node getNodeSymlinksUnderFolder(String contentUUID, String folderPath, String workspace) {

```

Reply:
done